design/ceph/object/store.md

```diff
@@ -119,3 +119,31 @@ Specifying this section also ensures that the pool section in the ceph-object-zo
   zone:
     name: "name"
 ```
+
+### LDAP
+
+The Ceph Object Gateway supports integrating with LDAP for authenticating and creating users, please refer [here](https://docs.ceph.com/docs/master/radosgw/ldap-auth/). This means that the `ceph object store user` is also required to be part of groups in the LDAP server otherwise, authentication will fail. The following settings need to be configured in the RGW server:
+```
```

Comment:
There is no mention of `rgw ldap searchfilter` in this block.  It isn't required to enable ldap support but I suspect it will be commonly used and a CRD should support it. Do we need to list the optional parameters in a separate subsection?